### PR TITLE
[ADD] Maintenance: add new equipment

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR creates a "Maintenance" section in the Manufacturing folder and adds documentation on adding new equipment to the Maintenance module.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @StraubCreative,

This is ready for review.

Thanks,
hojo

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+=================
+Add new equipment
+=================
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:3]: line break after heading
```suggestion
=================
Add new equipment
=================

```

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Using Odoo Maintenance, it is possible to track individual pieces of equipment, along with
+information about their maintenance requirements. To add a new piece of equipment, navigate to the
+:guilabel:`Maintenance` module, select :menuselection:`Equipments --> Machines & Tools --> Create`,
+and configure the equipment as follows:
+
+- :guilabel:`Equipment Name`: the product name of the piece of equipment
+- :guilabel:`Equipment Category`: the category that the equipment belongs to
+- :guilabel:`Company`: the company that owns the equipment
+- :guilabel:`Used By`: Specify if the equipment is used by a specific employee, department, or both.
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:12]: lowercase to match formatting in the rest of the list.

also, examples/use cases here?
when's it better to assign a person vs a department? how do you create/edit a department?
```suggestion
- :guilabel:`Used By`: specify if the equipment is used by a specific employee, department, or both.
```

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
+=================
+Add new equipment
+=================
+Using Odoo Maintenance, it is possible to track individual pieces of equipment, along with
+information about their maintenance requirements. To add a new piece of equipment, navigate to the
+:guilabel:`Maintenance` module, select :menuselection:`Equipments --> Machines & Tools --> Create`,
+and configure the equipment as follows:
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:7]: - italicize app name when not being included in a `menuselection` tag
- define _equipment_ and provide some context around it
```suggestion
Using Odoo *Maintenance*, it is possible to track individual pieces of equipment, along with
information about their maintenance requirements.

*Equipment* in this context refers to tools and machinery that's used on manufacturing or
engineering production lines, which can be owned by the company that uses the Odoo database, or a
third-party source such as a vendor in the case of equipment rentals.

To add a new piece of equipment, navigate to the :guilabel:`Maintenance` module, select
:menuselection:`Equipments --> Machines & Tools --> Create`, and configure the equipment as follows:
```

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+- :guilabel:`Equipment Name`: the product name of the piece of equipment
+- :guilabel:`Equipment Category`: the category that the equipment belongs to
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:10]: Such as...?
Create/edit new equipment categories by doing x, y, z.

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- :guilabel:`Company`: the company that owns the equipment
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:11]:
```suggestion
- :guilabel:`Company`: the company that owns the equipment. Again this can be the company that's
  listed as the Odoo database owner, or it can be a vendor in the case of equipment rentals.
```

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- :guilabel:`Used By`: Specify if the equipment is used by a specific employee, department, or both.
+  Select :guilabel:`Other` to specify both an employee and a department.
+- :guilabel:`Maintenance Team`: the maintenance team responsible for servicing the equipment
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:14]: What is a maintenance team? How do you set one up? What's a good or common use-case example?

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+- :guilabel:`Technician`: the specific technician responsible for servicing the equipment
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:15]: How are technicians created/assigned?

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+- :guilabel:`Used in location`: the location where the equipment is used
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:16]: such as...

File: content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+- :guilabel:`Work Center`: the work center where the equipment is used
+
+.. image:: add_new_equipment/new_equipment_form.png
```
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/manufacturing/maintenance/add_new_equipment.rst:19]: images get hyphens, not underscores as well as unique names.

```suggestion
.. image:: add_new_equipment/new-equipment-form.png
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @hojo-odoo 
Pausing here. I'd like this to be fleshed out more.
This super comes off like a UI/feature dump and doesn't have enough context around the fields, use case(s), 2nd and 3rd order tasks, etc. I find myself asking more questions than absorbing the information; the information is too lean and assumes the user knows too much. Please take another pass with an _ELI5_ narrative lens.

**Comment by hojo-odoo** [comment]
Hey @StraubCreative,

I've addressed all of your comments. This is ready for another review.

Thanks,
hojo

**Review by StraubCreative — APPROVED** [review]
LGTM! Nice work @hojo-odoo 
Quick rebase on [7af167d](https://github.com/odoo/documentation/pull/3532/commits/7af167db73568a5f005a25dcb32c74f355940d81). All checks pass ✅ 

@robodoo r+
